domains/jamied132.json

```
@@ -6,6 +6,7 @@
     "email": "jamied132.email@gmail.com"
   },
   "record": {
-    "CNAME": "jamied132.github.io"
+    "CNAME": "jamied132.github.io",
+    "NS:["kevin.ns.cloudflare.com","megan.ns.cloudflare.com"]
```

You only need NS records.
```suggestion
    "NS":["kevin.ns.cloudflare.com","megan.ns.cloudflare.com"]
```